### ISSUE TITLE
No README or config.properities dummy file

### ISSUE THREAD

**Issue by K33TY** [apl-cornell/fabric.ide#1]
I haven't figured out how to build this yet, but from looking at it:

It seems it requires installing jif ide first? (from looking at common.xml)

Would be useful for other users if dummy config.properties.in existed so that they know which paths to set. Looking at common.xml, looks like may need paths to this directory, jif.ide directory, eclipse directory?

**Comment by liujed** [comment]
What happens when you run `ant`?

**Comment by K33TY** [comment]
I couldn't find eclipse plugins folder on my mac...

Seems it was a hidden folder, you have to right click eclipse application and then "show package contents", and from there I was able to find the parent folder containing the plugins directory. 

Creating a variable `eclipse.home={above mentioned path}` helped this issue, but now getting an error regarding JIF IDE, so I guess i should install that first:

> ${fabric.ide}/common.xml:153: Failed to find Jif IDE. Set the 'jif-ide.home' property in ${fabric.ide}/config.properties, and make sure Jif IDE is built.

(looks like jif ide has a config.properties.in file)

However, it fails at this point: 

> ${jif.ide}/common.xml:152: Failed to find Polyglot IDE. Set the 'polyglot-ide.home' property in ${jif.ide}/config.properties, and make sure Polyglot IDE is built.

(So I guess I have to get this from the apl git, but assuming should also be here in that case?)

**Comment by liujed** [comment]
As @andrewcmyers  mentioned in Slack, the Polyglot IDE is hosted at https://github.com/polyglot-compiler/polyglot.ide/. Would be great if we can have that repo automatically mirror under the apl-cornell org for convenience.

**Comment by tmagrino** [comment]
@liujed so just maintain a mirrored fork of the (polyglot ide)[https://github.com/polyglot-compiler/polyglot.ide/] repo under this github org?  I can look into that.  It's a bit more difficult if we want to maintain mirrors that are both independently pushed to by developers.

**Comment by liujed** [comment]
Yeah, not clear what the right approach is. Another possibility would be to use submodules so people don't have to go hunting for dependencies. Not really sure if that's the Eclipse way of doing things, though.
